tests/lib/rules/indent.js

```diff
@@ -1428,7 +1441,7 @@ ruleTester.run("indent", rule, {
         {
             code: fixture,
             output: fixedFixture,
-            options: [2, {SwitchCase: 1}],
+            options: [2, {SwitchCase: 1, MemberExpression: 1}],
```

Comment:
hm, could one of you create an issue about it? I think it would be good if our code should be as consistent as possible so it could be kinda of an example how things should look like?